demos/list.scss

```diff
@@ -19,3 +19,107 @@
 @import "../packages/mdc-form-field/mdc-form-field";
 @import "../packages/mdc-list/mdc-list";
 @import "../packages/mdc-typography/mdc-typography";
+
+a.material-icons {
+  text-decoration: none;
+}
+
+.mdc-list,
+.mdc-list-group {
+  max-width: 600px;
+}
+
+.mdc-theme--dark {
+  background: #303030;
+}
+
+.mdc-theme--dark > h3 {
+  color: white;
+}
+
+.demo-list--icon-placeholders,
+.mdc-list--avatar-list {
+  @include mdc-list-item-icon-ink-color(white);
+  @include mdc-list-item-icon-fill-color(text-icon-on-background);
+
+  @include mdc-theme-dark {
+    @include mdc-list-item-icon-ink-color(#303030);
+    @include mdc-list-item-icon-fill-color(text-icon-on-dark);
+  }
+}
+
+.demo-list--meta-placeholders {
+  .mdc-list-item__end-detail {
+    background-color: currentColor;
```

Comment:
Is this not expressible via a mixin now? (Relevant to another PR that I think you merged?)

Reply:
I don't have a mixin for `background-color` on `__end-detail`, because I didn't see any mention of it in the spec.

And you're right that `mdc-theme-prop` now supports `currentColor` values; I just didn't see a need to use that mixin here since the value is hard-coded and not programmatically- or user-modifiable.